tests/unit/src/test/scala/tests/markdown/ResetSuite.scala

```diff
@@ -58,12 +59,15 @@ class ResetSuite extends BaseMarkdownSuite {
        |// x: Int = 42
        |```
        |
-       |```scala mdoc:reset
+       |```scala
+       |
        |```
        |
        |```scala
+       |implicit val x: Int = 41
```

Comment:
@hmf Can you elaborate on why this test was passing before? The `mdoc:reset` modifier should reset the scope so the `println(x)` should not compile. I updated the test case to introduce a new `implicit val x` value

Reply:
Context https://github.com/scalameta/mdoc/pull/192